Stdin/README.md

```diff
@@ -0,0 +1,123 @@
+# Stdin Interface
+
+**Status**: Proposed
+
+| Name      | ID                                                                |
+| :-------- | :---------------------------------------------------------------- |
+| Stdin     | 8796536366ee21852db56dccb60bc564598b618c865fc50c8b1ab740bba128e3  |
+
+
+## Description
+
+Allows to get input from user for basic types like integers, addresses, public keys, strings, cells.
+
+## Functions
+
+`getStr` - prints prompt message to the user and returns string entered by user
```

Comment:
Suggest to rename `get` to `input` as a more clear meaning for the user input functions. The `get` is usually used as a property getter function.

Reply:
Ok